### DISCUSSION TITLE
How does Vuetify do this?

### DISCUSSION THREAD

**Discussion by 9mm** [vuetifyjs/vuetify#16515] [Q&A]
I notice both Vuetify and MaterialUI have an awesome feature of their autocomplete inputs... when you click an item, the input does NOT lose focus.

I'm building my own autocomplete feature (not Vuetify) and I would love to add this.

I've looked online for various solutions on how to make a click event NOT let an input blur, and all the solutions have their own issues, ie:
* use `mousedown`/`touchstart` instead of `click`... this prevents scrolling the autocomplete with your finger
* re-focus the input... this has a lot of issues, namely firing unwanted blur events

I've looked through the source of Vuetify and it's not immediately clear how this is being done... https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts

**Comment by KaelWD** [comment]
preventDefault mousedown on the list items

**Reply by 9mm** [reply]
awesome thanks, I actually tried that but I combined it with `touchStart` and so shot myself in the foot. Thank you!
